### ISSUE TITLE
Apple App verification fails.

### ISSUE THREAD

**Issue by gcorfield** [CocoaLumberjack/CocoaLumberjack#1077]
### New Issue Checklist

* [x ] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [ x] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [ x] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name       osx    | e.g. ios / osx / tvos / watchos
 Platform Version     osx10.14.4   | e.g. 8.0
 CocoaLumberjack Version see attached image| e.g. 2.3.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version        Xcode 10.1   | e.g. Xcode 7.3
 Repro rate       100%       | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps
I work for TalkTalk App development. We are building our first beta.
After I upload my app using cocoaLumberjack I get this from Apple...
Dear Developer,

We identified one or more issues with a recent delivery for your app, "MyTalkTalk" 4.0.98 (4.0.98). Please correct the following issues, then upload again.

ITMS-90338: Non-public API usage - The app contains or inherits from non-public classes in MyTalkTalk: DDLog . If method names in your source code match the private Apple APIs listed above, altering your method names will help prevent this app from being flagged in future submissions. In addition, note that one or more of the above APIs may be located in a static library that was included with your app. If so, they must be removed. If you think this message was sent in error and that you have only used Apple-published APIs in accordance with the guidelines, send the app's Apple ID, along with detailed information about why you believe the above APIs were incorrectly flagged, to appreview@apple.com. For further information, visit the Technical Support Information at http://developer.apple.com/support/technical/

Best regards,

The App Store Team
Any help would be appreciated

<img width="758" alt="Screenshot 2019-06-19 at 15 50 02" src="https://user-images.githubusercontent.com/4058935/59776149-3b63c280-92aa-11e9-82c6-0f32124193f0.png">

Please fill in the detailed description of the issue (full textual output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

Are you reporting a queue deadlock? If so, you must include a complete backtrace of all threads, which you can generate by typing `bt all` in the debugger after you hit the deadlock. Failure to do so will prevent us from helping you.

**Comment by sushichop** [comment]
@gcorfield 
Thanks for reporting this issue. Unfortunately I don't know why Apple verification fails for now.
Even now, do you still get the same problem using the latest Xcode(10.2.1) and CocoaLumberjack(3.5.3)?

**Comment by bpoplauschi** [comment]
@gcorfield any response or update to this issue?

**Comment by gcorfield** [comment]
Issue was resolved by removing framework from Apple upload. Not sure why it was failing. Thank you
